src/wasm/wasm-s-parser.cpp

```diff
@@ -1650,20 +1650,36 @@ void SExpressionWasmBuilder::parseImport(Element& s) {
     }
   } else if (im->kind == ExternalKind::Table) {
     if (j < inner.size() - 1) {
-      wasm.table.initial = atoi(inner[j++]->c_str());
+      uint64_t num = atoi(inner[j++]->c_str());
```

Comment:
4x reuse, how about extracting this to `uint64_t checkedTableSize(char*, Element&);`

And/or, they all say "excessive table size", when two of them are memories.